tests/ubiconfig/loaders/test_loader.py

```diff
@@ -9,6 +9,12 @@ def test_no_load():
         Loader().load('something.yaml')
 
 
+def test_no_load_by_cs_label():
```

Comment:
This test is passing because we are calling load_by_cs_label from Loader in base class, which calls load_all, from base class too, and that's the method that is raising the NotImplementedError. So the test is asserting that we cannot call load_by_cs_label from base class. Can you update the doc string?